tests/test_browser_model.py

```diff
@@ -49,6 +51,7 @@ def test_switch_to(foxpuppet, selenium):
 def test_tracking_protection_shield(foxpuppet, selenium):
     """Tests if the tracking protection icon displays."""
     browser = foxpuppet.browser
+    time.sleep(5)  # it seems necessary to sleep before navigating to get the shield
```

Comment:
Looks like they maybe fetch some data after first start, or update the database. Maybe just ask Johann Hofmann if he knows. If yes, there might be a way to circumvent this.

Reply:
Ah, and don't you use a call to wait here?

Reply:
Yes, this is my guess too. You'll notice that I closed this pull request though, as it didn't fix the issue. It looks like the 'regression' may be Linux only, so I'll dig into it a little more today.

Reply:
Ok, let me know.